README.md

```diff
@@ -1,45 +1,151 @@
-# My Portfolio
-> Click [here](https://github.com/josman227/Portfolio-Site) to access the webpage
-> 
->This front end design portfolio is an implementation of figma design tools
->Flexbox is used to arrange elements in positions in this page 
-## Built With
+<a name="readme-top"></a>
 
-- HTML5
-- CSS3
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+ 
+
+  <h3><b>Mobile First Portfolio Site</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Mobile First Portfolio Project] <a name="about-project"></a>
+
+**[Hello Microverse]** is the first Microverse project to setup Linters for the best coding standards
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li>HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+
+### Key Features <a name="key-features"></a>
+
+- **[HTML Hint]**
+- **[CSS LINT]**
+
```

Comment:
Please close the details tag here. It is causing a bug in your readme file where you can see what comes after line 52. 👍